src/js/components/List/List.js

```diff
@@ -0,0 +1,222 @@
+import React from 'react';
+import { compose } from 'recompose';
+import styled, { withTheme } from 'styled-components';
+
+import { Box } from '../Box';
+import { InfiniteScroll } from '../InfiniteScroll';
+import { Keyboard } from '../Keyboard';
+import { Text } from '../Text';
+import { focusStyle } from '../../utils';
+import { withFocus, withForwardRef } from '../hocs';
+
+const StyledList = styled.ul`
```

Comment:
should we allow `ul` vs `ol` to be configurable?

Reply:
The caller can use `as="ol"` to change if needed. Although, I don't think there are any differences between the two in our case. In vanilla HTML, `ol` gets numeric bullets and `ul` gets solid disc bullets, neither of which apply for us here.